database/queries/getUserByGoogleID.js

```diff
@@ -0,0 +1,5 @@
+module.exports = 
+
+`SELECT * 
+FROM user 
+WHERE GoogleID = ?`
```

Comment:
We have separate files for these SQL queries...just better encapsulated. It'll be referred to within the dbService.js